app/lib/navigation/state_file_md_question_navigation.rb

```diff
@@ -5,6 +5,7 @@ class StateFileMdQuestionNavigation < Navigation::StateFileBaseQuestionNavigatio
     SECTIONS = [
       Navigation::NavigationSection.new("state_file.navigation.section_1", [
         Navigation::NavigationStep.new(StateFile::Questions::MdEligibilityFilingStatusController),
+        Navigation::NavigationStep.new(StateFile::Questions::EligibilityOffboardingController, false),
```

Comment:
my first initial solve was to add this back, but realized that this was not the full extent of the issue. All other state navigations have the EligibilityOffboardingController in the navigation -- in the odd case this controller does not have an `offboarded_from` param, it will default to the `form_navigation.prev` to determine what to set the path for "Go back to correct" link.